### PR TITLE
feat: bootstrap @invisible eslint plugin

### PR DESCRIPTION
**Pull request by rodrigo4244**

### CODE REVIEW

**Comment by rodrigo4244** [comment]
For testing you can use:
https://github.com/invisible-tech/merge-parsers/pull/12

File: README.md [hunk 1]
```diff
@@ -1,19 +1,32 @@
...
+2. Install the devDependencies below:
+```
+yarn add -D eslint
+yarn add -D eslint-config-airbnb
+yarn add -D eslint-formatter-pretty
+yarn add -D eslint-plugin-import
+yarn add -D eslint-plugin-jsx-a11y
+yarn add -D eslint-plugin-lodash-fp
+yarn add -D eslint-plugin-promise
+yarn add -D eslint-plugin-react
```
Comment (gunar) [inline README.md]: These lines should be replaced by
```
yarn add -D eslint @invisible/eslint-config
```

**Comment by gunar** [comment]
This is what I want the consumer to look like. If I install `@invisible/eslint-config`, all plugins should come with it. The consumer should not have to install plugins manually.

> https://github.com/invisible-tech/merge-parsers/pull/12
> https://github.com/invisible-tech/merge-parsers/pull/12/commits/3cfb5180554fe21a4219bc581714a0f8c8813680

You might be able to use this as an example:
> https://github.com/janis-kra/eslint-config-janiskra

**Comment by rodrigo4244** [comment]
For testing you can use:
https://github.com/invisible-tech/merge-parsers/pull/12

**Comment by gunar** [comment]
WOOO it works!

NB: You squashed new commit into the "Update README". You should not have done that. We don't modify commits after review has began.

**Pull request merged**